Adjust Preferences (#579)
iterate-ch/docs: mountainduck/preferences.md

Save all mounted volumes when quitting to be restored while relaunching.

> If you enable *Save Workspace* and [*Enable Login Item*](#login-item) in the same named preferences tab and do not manually eject the volume prior to reboot it will reconnect after login.

Reviewer: ```suggestion
If you enable *Save Workspace* and [*Enable Login Item*](#login-item) within *Preferences → Login Item* and do not manually eject the volume prior to quitting Mountain Duck, it will reconnect after login.
```

## Login Item

Reconnect after restarting the computer. 

> If you enable *Enable Login Item* and [*Save Workspace*](#save-workspace) in *Preferences → General* and do not manually eject the volume prior to reboot it will reconnect after login.

Reviewer: ```suggestion
If you enable *Enable Login Item* and [*Save Workspace*](#save-workspace) in *Preferences → General* and do not manually eject the volume prior to quitting Mountain Duck, it will reconnect after login.
```

## Hidden Configuration Options

> These settings are shared with Cyberduck.

Reviewer: remove line, superseeded by PR #577
